Source/quests.h

```diff
@@ -51,7 +51,7 @@ struct QuestStruct {
 	dungeon_type _qlvltype;
 	Point position;
 	_setlevels _qslvl;
-	uint8_t _qidx;
+	quest_id _qidx;

```

Comment:
Could even rename this _id or something similar, it was only ever used to index into the array once in DRLG_CheckQuests which is unnecessary now.

Reply:
Yeah, most enums needs some heavy renaming so I'm half way pushing it in to next week, halfway hoping that others feel like they want to rename some of them.